### PR TITLE
Update for hc call zome

### PR DESCRIPTION
**Pull request by willemolding**
hClient now exposes callZome to maintain compatibility with updates to hc-web-client

### CODE REVIEW

**Comment by willemolding** [comment]
Closes #21

**Comment by maackle** [comment]
@willemolding it still feels awkward to me that hClient expects the client to know about the instanceId. I guess it needs to be that way to make it a true drop-in replacement for hc-web-client?

Maybe we can expand this with a very flexible input. Like, you can pass in `"instance/zome/func"`, or an object with keys `instanceId`, `zome`, and `function`. OR, an object with `agentId`, `dnaHash`, `zome`, `function`? Thoughts?

File: src/index.js [hunk 1]
```diff
@@ -144,6 +144,11 @@ const hClient = (function () {
             const { callString: newCallString, params: newParams } = await preCall(dnaHash, callString, params)
             return call(newCallString)(newParams).then(postCall)
           },
+          callZome: (instanceId, zome, func) => async (params) => {
```
Comment (maackle) [inline src/index.js]:
```suggestion
          callZome: (_instanceId, zome, func) => async (params) => {
```

File: src/index.js [hunk 2]
```diff
@@ -144,6 +144,11 @@ const hClient = (function () {
             const { callString: newCallString, params: newParams } = await preCall(dnaHash, callString, params)
             return call(newCallString)(newParams).then(postCall)
           },
+          callZome: (_instanceId, zome, func) => async (params) => {
+            const callString = [instanceId, zome, func].join('/')
```
Comment (willemolding) [inline src/index.js]:
```suggestion
            const callString = [_instanceId, zome, func].join('/')
```

Comment (maackle) [inline src/index.js]:
```suggestion
            const callString = ["IGNORED", zome, func].join('/')
```

**Pull request merged**